utils/src/main/scala/com/salesforce/op/utils/spark/OpSparkListener.scala

```diff
@@ -129,6 +143,23 @@ class OpSparkListener
 
 }
 
+trait MetricJsonLike extends JsonLike {
+  override def toJson(pretty: Boolean): String = {
+    JsonUtils.toJsonString(this, pretty = pretty, Seq(SerDes[Max[Long]](
+      classOf[Max[Long]],
+      new StdSerializer[Max[Long]](classOf[Max[Long]]) {
+        override def serialize(
+          value: Max[Long],
+          gen: JsonGenerator, provider: SerializerProvider
+        ): Unit = {
+          gen.writeNumber(value.get)
```

Comment:
do you need getOR else there? Can it be null?

Reply:
it's not an option. It's a function unwrapping `Long` from `Max[Long]` semigroup